tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -228,6 +228,28 @@ Given('the administrator has excluded group {string} from sharing', async functi
   }
 })
 
+Given('the administrator has excluded group {string} from receiving shares', async function (group) {
+  const configList = await runOcc([
+    'config:list'
+  ])
+  const config = _.get(configList, 'ocs.data.stdOut')
+  const configParsed = JSON.parse(config)
+  const initialExcludedGroup = JSON.parse(_.get(configParsed, 'apps.files_sharing.blacklisted_receiver_groups') || '[]')
+  if (!initialExcludedGroup.includes(group)) {
+    initialExcludedGroup.push(group)
+    const resultGroupList = initialExcludedGroup.map((res) => '"' + res + '"')
+    const resultToString = resultGroupList.join(',')
```

Comment:
The variable name is vague. You can just use the same name instead (this is a dynamic langauge :slightly_smiling_face: )
```suggestion
    excludedGroups = excludedGroups.join(',')
```